packages/widget_toolkit_pin/example/lib/main.dart

```diff
@@ -111,35 +111,100 @@ class MyHomePage extends StatelessWidget {
           dialogHasBottomPadding: true,
         ),
       );
+
+  String _exampleMapMessageToString(BiometricsMessage message) {
+    switch (message) {
+      case BiometricsMessage.notSetup:
+        return 'To use biometrics, you need to turn it on in your device settings!';
+
+      case BiometricsMessage.notSupported:
+        return 'You don\'t have biometric feature on your device!';
+
+      case BiometricsMessage.enabled:
+        return 'Your biometrics are enabled!';
+
+      case BiometricsMessage.disabled:
+        return 'Your biometrics are disabled!';
+    }
+  }
 }
 
 class AppPinCodeService implements PinCodeService {
-  AppPinCodeService();
+  AppPinCodeService({required this.sharedPreferences});
```

Comment:
Please keep the sample as simple as possible. For example here you can use in memory storage instead of saving values in shared preferences

Reply:
I tried to replace shared preferences with in memory storage, but on app restart the the method `isPinCodeInSecureStorage` will not return the pin code, which will not be saved at this time any more and also
the method `areBiometricsEnabled` will not return true and the auto biometric authentication will not work for the example.


 ``` @override
  Future<bool> isPinCodeInSecureStorage() {
    var isPinCodeInSecureStorage =
        inMemoryInstance.getString(_isPinCodeInStorage);
        
        
Future<bool> areBiometricsEnabled() async {
    var areBiometricsEnabled = inMemoryInstance.getBool(_areBiometricsEnabled);      ```

Reply:
I hava replaced the shared preferences example with in memory storage example